sensing/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Tier IV, Inc.
+// Copyright 2020-2023 Foxconn, Tier IV, Inc.
```

Comment:
I apologize, but since there have been no modifications to this source file.
Could you please revert the copyright back to its original state?

Reply:
Ok. Done.

Reply:
Thank you!